test/Test/Database/LSMTree/Normal/StateMachine.hs

```diff
@@ -1323,6 +1323,15 @@ data Stats = Stats {
     -- | Size of tables that were closed (to augment the size of the one
     -- remaining open at the end of the run).
   , closedTableSizes   :: !(Map Model.TableHandleID Int)
+    -- | The ultimate parent for each table. This is the 'TableId' of table
+    -- created using 'new' or 'open'.
+  , parentTable        :: Map Model.TableHandleID Model.TableHandleID
+    -- | Track the interleavings of operations via different but related tables.
+    -- This is a map from the ultimate parent table to a summary log of which
+    -- tables (derived from that parent table via duplicate) have had actions
+    -- performed on them. We record only the interleavings of different tables
+    -- not multiple actions on the same table.
+  , dupTableActionLog  :: Map Model.TableHandleID [Model.TableHandleID]
```

Comment:
Might be useful to mention here (and/or in other places) that we only count the operations that we consider to be interesting